scripts/config.py

```diff
@@ -265,35 +262,37 @@ def get_supported_GCC(self):
         # we need GCC with C++17 support
         # https://gcc.gnu.org/projects/cxx-status.html
         return ['',
-                11.3, 11.2, 11.1, 11,
+                11.4, 11.3, 11.2, 11.1, 11,
                 10.4, 10.3, 10.2, 10.1, 10,
                 9.5, 9.4, 9.3, 9.2, 9.1, 9,
-                8.5, 8.4, 8.3, 8.2, 8.1, 8,
-                7.5, 7.4, 7.3, 7.2, 7.1, 7]
+                8.5, 8.4, 8.3, 8.2, 8.1, 8]
 
-    def _set_compiler_linker_helper(self, opt, prg, versions):
+    def _set_compiler_linker_helper(self, opt, prg, versions, show=False):
         if not self.is_empty(opt):
             return
-        prg = find_newest(prg, versions, True)
+        prg = find_newest(prg, versions, show=show)
         if isCIBuild() and prg:
             prg = 'ccache ' + prg
         self._set(opt, prg)
 
     def _set_cxx(self):
         self._set_compiler_linker_helper(
-            Config.KEY_CXX, 'g++', self.get_supported_GCC())
+            Config.KEY_CXX, 'g++', self.get_supported_GCC(), show=True)
 
     def _set_linker(self):
         self._set_compiler_linker_helper(
             Config.KEY_LINKERFORPROGRAMS, 'g++', self.get_supported_GCC())
 
     def _config_compiler(self):
+        print('Configuring compiler')
         if self.configDict["DEBUG"] == "":
             self.configDict["DEBUG"] = "False"
-
-        if self.configDict["CC"] == "" and checkProgram("gcc"):
+        if self.configDict["CC"] == "" and checkProgram("gcc")[0]:
             self.configDict["CC"] = "gcc"
-            print(green('gcc detected'))
+            if get_GCC_version("gcc")[0] < Config.MINOR_GCC_VERSION:
+                print(red("gcc version required >= 8, detected gcc {}".format(get_GCC_version("gcc")[0])))
```

Comment:
Variable for minimum gcc version, so we do not need to manually change it each time it gets updated